attributes/default.rb

```diff
@@ -30,6 +30,7 @@
 default[:mongodb][:replica_priority] = 1
 default[:mongodb][:replica_tags] = {}
 default[:mongodb][:replica_votes] = 1
+default[:mongodb][:add_by_ip] = false
```

Comment:
Can you change this to `replset_add_by` and set the default to `fqdn`?  Then you can use this below like so: `members[n][node[:mongodb][:replset_add_by]]`